internal/cli/users.go

```diff
@@ -22,8 +67,341 @@ func usersCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listUsersCmd(cli))
+	cmd.AddCommand(createUserCmd(cli))
+	cmd.AddCommand(showUserCmd(cli))
+	cmd.AddCommand(deleteUserCmd(cli))
+	cmd.AddCommand(updateUserCmd(cli))
 	cmd.AddCommand(userBlocksCmd(cli))
 	cmd.AddCommand(deleteUserBlocksCmd(cli))
+
+	return cmd
+}
+
+func listUsersCmd(cli *cli) *cobra.Command {
```

Comment:
Thanks for doing the full CRUD!
We cannot really have `users list` as it will only show the first X users. Can you please turn this into `users search`? https://auth0.com/docs/api/management/v2#!/Users/get_users

Flags:
**query** (string) _Required_
Long form: `query`
Short form: `q`
Help: Query in Lucene query string syntax. See https://auth0.com/docs/users/user-search/user-search-query-syntax for more details.

**sort** (string) -> should not have `Always prompt` set to `true`
Long form: `sort`
Short form: `s`
Help:  Field to sort by. Use `field:order` where `order` is `1` for ascending and `-1` for descending. e.g. `created_at:1`.